_opensearch/query-dsl/index.md

```diff
@@ -12,16 +12,66 @@ redirect_from:
 
 # Query DSL
 
-While you can use HTTP request parameters to perform simple searches, you can also use the OpenSearch query domain-specific language (DSL), which provides a wider range of search options. The query DSL uses the HTTP request body, so you can more easily customize your queries to get the exact results that you want.
+OpenSearch provides a query domain-specific language (DSL) that you can use to perform customized searches on your data.You can even perform more complicated searches by combining query fields to find more granular data that you need.
 
-For example, the following request performs a simple search to search for a `speaker` field that has a value of `queen`.
+OpenSearch supports the following query DSL query type categories:
+
+- **Compound queries** – To perform combined queries with the following query types:
+    - **Boolean** `bool` – Combines several simple query terms by boolean clause rules that act as logical operators. To learn more, see [Boolean queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/bool/).
+    - **Constant score** `constant_score` – Provides a `filter` query with the relevance score as a floating point number assigned by the `boost` field. Acts as a wrapper for the filter.
+    - **Disjunction max** `dis_max` – Returns documents that match one or more queries that are wrapped together by clauses. If a document matches multiple query clauses, it gets the highest relevant score assigned to it.
+    - **Function score** `function_score` – Modifies the score of documents that are returned by a query. You define a query and one or more functions that each compute a score for each document that matches the query.
+
+- **Full-text queries** – To search documents for one or more terms and filter with advanced options. These queries return detailed information including relevance scores per document match, and perform text analysis. To learn more, see [Full-text queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/full-text/).
+
+- **Geographic and shape queries** – To create queries to search two-dimensional geometries that map out cartesian data such as `point` fields that support x and y pairs and `shape` fields that support points, lines, circles and polygon shapes. To learn more, see [Geographic and shape queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/geo-and-shape/).
+
+- **Joining queries** – To perform `nested`, `has_child`, `has_parent`, and `parent_id` queries.
+
+- **Parameter queries** – To perform `minimum_should_match` and `rewrite` queries.
```

Comment:
There are no "Parameter queries" as such. `minimum_should_match` and `rewrite` are parameters that can be used in queries. We can either talk about the parameters in a different list, or remove this line altogether.